src/templates/expressions/typescriptfieldtypeconverter.ts

```diff
@@ -0,0 +1,42 @@
+const KintoneStringFieldTypeAndTypeScriptFieldTypeName = {
```

Comment:
Now that you merge all field types into one object, is the term "String" (used in `KintoneString...` ) still necessary?